### PR TITLE
Update Unicode Version to 8.0.0

### PR DESCRIPTION
**Pull request by sharmaansh21**

### CODE REVIEW

**Comment by sgrif** [comment]
Can you please give context about why this change is needed in your commit message?

**Comment by sgrif** [comment]
This also caused some test failures that look legitimate.

**Comment by rafaelfranca** [comment]
You need to run `bin/generate_tables` at activesupport folder and commit the generated file too.

**Comment by justanshulsharma** [comment]
@sgrif everything is green now.

@rafaelfranca :+1:

**Comment by carlosantoniodasilva** [comment]
@justanshulsharma can you add a changelog entry as well? In case it causes any issue to anyone using the framework, that's probably the first place people will look at. Please amend your commit to include that, thanks!

**Comment by justanshulsharma** [comment]
@carlosantoniodasilva Done :+1:

File: guides/CHANGELOG.md [hunk 1]
```diff
@@ -18,4 +18,8 @@
 
     *Andrey Nering*
 
+*   Updated Unicode version to 8.0.0
+
+    *Anshul Sharma*
```
Comment (carlosantoniodasilva) [inline guides/CHANGELOG.md]: Oh, I forgot to mention, but this should be in the activesupport changelog instead (there is where the change happened, right?). Also you can put the entry on the top :)

Comment (justanshulsharma) [inline guides/CHANGELOG.md]: yes you are right, i fixed it now @carlosantoniodasilva

**Comment by carlosantoniodasilva** [comment]
Thanks!

**Pull request merged**